src/libPMacc/PMaccConfig.cmake

```diff
@@ -167,41 +155,93 @@ if(VAMPIR_ENABLE)
         -v)
 
     # for manual instrumentation and hints that vampir is enabled in our code
-    add_definitions(-DVTRACE)
+    set(PMacc_DEFINITIONS ${PMacc_DEFINITIONS} -DVTRACE)
 
     # titan work around: currently (5.14.4) the -D defines are not provided by -vt:showme-compile
-    add_definitions(-DMPICH_IGNORE_CXX_SEEK)
+    set(PMacc_DEFINITIONS ${PMacc_DEFINITIONS} -DMPICH_IGNORE_CXX_SEEK)
 endif(VAMPIR_ENABLE)
 
+
 ################################################################################
-# MPI LIB
+# Find MPI
 ################################################################################
-find_package(MPI MODULE REQUIRED)
+
+find_package(MPI REQUIRED)
```

Comment:
[good short documentation `CONFIG` vs. `MODULE` ](https://github.com/ruslo/hunter/wiki/example.find_package.config)

If we switch to modern cmake target we should use ĆONFIG`